docs/resources/samples/add-excel-comments.md

```diff
@@ -67,4 +67,4 @@ function main(workbook: ExcelScript.Workbook) {
 
 ## Training video: Add comments
 
-[![Watch step-by-step video on how to add comments in an Excel file](../../images/comments-vid.jpg)](https://youtu.be/CpR78nkaOFw "Step-by-step video on how to add comments in an Excel file")
+[Watch Sudhi Ramamurthy walk through this sample on YouTube](https://youtu.be/CpR78nkaOFw).
```

Comment:
Could we use the video embed format? e.g. [!VIDEO https://www.youtube.com/embed/vPEqbb7t6-Y?start=154] (I got this from the "before" on docs/resources/scenarios/analyze-web-downloads.md)

Reply:
@mscharlock gave the guidance to not embed YouTube videos, as they do not meet accessibility criteria. Also, these videos are ostensibly from a third-party, so we did not want to promote them as first-party content.

Reply:
PR #269 addresses this. My apologies for merging the two here.